internal/accounts/create.go

```diff
@@ -134,3 +162,158 @@ func create(
+		signer, err := state.EmulatorServiceAccount()
+		if err != nil {
+			return nil, err
+		}
+		account, err := service.Accounts.Create(
+			signer,
+			[]crypto.PublicKey{key.PublicKey()},
+			[]int{flow.AccountKeyWeightThreshold},
+			[]crypto.SignatureAlgorithm{crypto.ECDSA_P256},
+			[]crypto.HashAlgorithm{crypto.SHA3_256},
+			nil,
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		log.Info("\nPlease note, that the newly created account will only be available until you keep the emulator service up and running, if you restart the emulator service all accounts will be reset. If you want to persist accounts between restarts you must use the '--persist' flag when starting the flow emulator.")
+
+		address = account.Address
+	} else {
+		var link string
+		switch network {
+		case config.DefaultTestnetNetwork():
+			log.Info("\nA testnet faucet website will open, follow the steps to create an account: \n 1. Fill in the captcha, \n 2. Click on 'Create Account' button.\n")
+			link = util.TestnetFaucetURL(key.PublicKey().String(), crypto.ECDSA_P256)
+		case config.DefaultMainnetNetwork():
+			log.Info("\nA Flow Port website will open, follow the steps to create an account: \n 1. Click on 'Submit' button, \n 2. Connect existing Blocto or create a new account first, \n 3. Click on confirm, \n 4. Click on approve. \n")
+			link = util.MainnetFlowPortURL(key.PublicKey().String())
+		}
+
+		time.Sleep(time.Second * 2)
+		err := util.OpenBrowserWindow(link)
+		if err != nil {
+			return nil, err
+		}
+
+		log.StartProgress("Waiting for an account to be created, please finish all the steps in the browser...")
+
+		addr, err := getAccountCreatedAddressWithPubKey(service, key.PublicKey(), startHeight)
+		if err != nil {
+			return nil, err
+		}
+		address = *addr
+
+		log.StopProgress()
+	}
+
+	var flowkitAccount *flowkit.Account
+	account, err := service.Accounts.Get(address)
+	if err != nil {
+		return nil, err
+	}
+
+	name := output.NamePrompt()
+
+	flowkitAccount, err = flowkit.AccountFromFlow(account, name, key)
+	if err != nil {
+		return nil, err
+	}
+	privateKeyEnvKey := "PRIVATE_KEY_" + strings.ToUpper(strings.TrimPrefix(address.String(), "0x"))
+	os.Setenv(privateKeyEnvKey, key.String())
+	log.Info(output.SuccessEmoji() + privateKeyEnvKey + " = " + strings.TrimPrefix(key.String(), "0x"))
+
+	if network == config.DefaultMainnetNetwork() && output.EnableSaveEnvPrompt() {
+		path, _ := os.Getwd()
+		envFilePath := path + "/private_keys.env"
+
+		f, err := os.OpenFile(envFilePath,
+			os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			fmt.Printf("Error with .env file: " + err.Error())
+		}
+		defer f.Close()
+		newPKeyEnvVar := privateKeyEnvKey + "=" + strings.TrimPrefix(key.String(), "0x")
+		if _, err := f.WriteString("\n" + newPKeyEnvVar); err != nil {
+			fmt.Printf("Error with writing to .env file:" + err.Error())
+		}
+		log.Info(output.SuccessEmoji() + "Private key env variable stored in cmd/flow/private_keys.env")
```

Comment:
Edited Gregor's implementation of using encryption for mainnet as discussed, and as intermediate solution to use environment variable to store private key, with the option to save in .env file if desired. 

Implementation here will save in the .env file in cmd/flow/private_key.env and will create the env file automatically if there is none detected in cmd/flow. If the file is detected, it will just add $PRIVATE_KEY_{address} = {privatekey} to the next line